pkg/scheduler/api/resource_info.go

```diff
@@ -368,6 +368,35 @@ func (r *Resource) LessEqualStrict(rr *Resource) bool {
 	return true
 }
 
+// LessInSomeDimension returns true if there exists any dimension whose resource amount in r is less than that in rr.
+// Otherwise returns false.
+// Note: Any dimension of resource, which is not listed in resource object, is regarded as zero.
+func (r *Resource) LessInSomeDimension(rr *Resource) bool {
```

Comment:
please add some unit-test case for this function

Reply:
OK, done.